test/tools/unified-spec-runner/schema.ts

```diff
@@ -386,6 +386,7 @@ export interface StoreEventsAsEntity {
 }
 export interface ExpectedError {
   isError?: true;
+  isTimeoutError?: boolean;
```

Comment:
I can't find where we assert the following logic, can you point me to it? 

> isTimeoutError: Optional boolean. If true, the test runner MUST assert that the error represents a timeout due to use of the timeoutMS option. If false, the test runner MUST assert that the error does not represent a timeout.

Reply:
We do not on main this is from the CSOT branch, I've reverted this, it will be added as a part of that branch.